.github/workflows/backfill-workflow-job.yml

```diff
@@ -10,14 +10,12 @@ defaults:
     working-directory: torchci
 jobs:
   backfill-rockset:
-    runs-on: ubuntu-20.04
+    runs-on: linux.large
     steps:
       - uses: actions/checkout@v3
       - run: yarn install --frozen-lockfile
       - run: yarn node scripts/backfillJobs.mjs
         env:
           ROCKSET_API_KEY: ${{ secrets.ROCKSET_API_KEY }}
-          OUR_AWS_ACCESS_KEY_ID: ${{ secrets.OUR_AWS_ACCESS_KEY_ID }}
```

Comment:
For this one, I think we should wait for Jean's PR to reland https://github.com/pytorch/test-infra/pull/4751 because using self-hosted runner here exposes other credentials `ROCKSET_API_KEY`, and `PRIVATE_KEY` to the non-ephemeral runners `linux.large`

Reply:
I don't understand @huydhn, this PR runs this job in a a GH provided runner and uses OIDC to authenticate to AWS. Reopening @clee2000 PR and merging if it passes after adding the permissions to run this workflow with OIDC on https://github.com/pytorch-labs/pytorch-gha-infra/pull/303

Reply:
Ah, my comment was about the previous version of the change before @clee2000 reworks this into the current state, which LGTM!